lxd/db/cluster/entity_type_identity.go

```diff
@@ -79,3 +82,25 @@ CREATE TRIGGER %s
 	END
 `, name, e.code(), typeCertificate.code(), e.code(), typeCertificate.code())
 }
+
+// authMethodCaseClause returns the SQL CASE clause for auth method mapping.
+func authMethodCaseClause() string {
+	return fmt.Sprintf(`CASE identities.auth_method
+		WHEN %d THEN '%s'
+		WHEN %d THEN '%s'
+	END`, authMethodTLS, api.AuthenticationMethodTLS, authMethodOIDC, api.AuthenticationMethodOIDC)
+}
+
+// typeInClause returns the SQL IN clause for the provided db type codes.
+func typeInClause(types ...int64) string {
+	if len(types) == 0 {
+		return "()"
+	}
+
+	strTypes := make([]string, len(types))
+	for i, t := range types {
+		strTypes[i] = strconv.FormatInt(t, 10)
+	}
+
+	return "(" + strings.Join(strTypes, ", ") + ")"
+}
```

Comment:
This function isn't really about identity types, it returns a string like `(1,2,3)` based on the input. I think it would be worth creating a util for it in the `lxd/db/query` package. It could look like:
```go
func IntParams[T int | int8 | int16 | int32 | int64 | uint | uint8 | uint16 | uint32 | uint64](args ...T) string {
	strs := make([]string, 0, len(args))
	for _, a := range args {
		if a > 0 {
			strs = append(strs, strconv.FormatUint(uint64(a), 10))
			continue
		}

		strs = append(strs, strconv.FormatInt(int64(a), 10))
	}

	return "(" + strings.Join(strs, ", ") + ")"
}
```